test/oqs_test_evp_pkey_params.c

```diff
@@ -46,6 +46,20 @@ const char *kHybridSignatureAlgorithms[] = {
 };
 ///// OQS_TEMPLATE_FRAGMENT_HYBRID_SIG_ALGS_END
 
+///// OQS_TEMPLATE_FRAGMENT_COMPOSITE_SIG_ALGS_START
+
+/** \brief List of composite signature algorithms. */
+const char *kCompositeSignatureAlgorithms[] = {
```

Comment:
Can I ask where this array is used?

Reply:
I was adding composite to the evp pkey test, I started by creating this array but did not implement the changes to do the testing yet.
I added this by mistake to this PR, since this work is not fully done yet.
I can remove it for now, if you want to.

Reply:
> I can remove it for now if you want to.

No extra work to clean this up, please, if you'll make use of this eventually: I see oqsprovider by now as a pure research test bed and it should serve that purpose as well as possible without hassles to contributors moving forward this purpose. I was just curious, so thanks for the explanation. Will also be curious how this is getting used eventually.